source/codegen/metadata/nirfmxpulse/config.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+config = {
+    'api_version': '25.0.0',
+    'c_header': 'niRFmxPulse.h',
+    'c_function_prefix': 'RFmxPulse_',
+    'service_class_prefix': 'NiRFmxPulse',
+    'java_package': 'com.ni.grpc.nirfmxpulse',
+    'csharp_namespace': 'NationalInstruments.Grpc.NiRFmxPulse',
+    'namespace_component': 'nirfmxpulse',
+    'close_function': 'Close',
+    'custom_types': [],
+    'additional_headers': {},
+    'type_to_grpc_type': {
+        "char[]": "string",
+        "float32": "float",
+        "float64": "double",
+        "int16": "int32",
+        "int32": "int32",
+        "int64": "int64",
+        "niRFmxInstrHandle": "nidevice_grpc.Session",
+        "int8": "int32",
+        "uInt16": "uint32",
+        "uInt32": "uint32",
+        "uInt64": "uint64",
+        "uInt8": "uint32",
+        "uInt8[]": "bytes",
+        "NIComplexSingle": "nidevice_grpc.NIComplexNumberF32",
+        "NIComplexDouble": "nidevice_grpc.NIComplexNumber",
+    },
+    'code_readiness': 'Release',
```

Comment:
```suggestion
    'code_readiness': 'NextRelease',
```
Change this to NextRelease until it's ready.

Reply:
@RahulBenaka , sorry I maybe shouldn't have put it as a suggested change you could commit directly to the branch. You'll need to build / regenerate again locally and push that up to fix the build.